climada/util/test/test_finance.py

```diff
@@ -86,17 +86,20 @@ def test_ne_gdp_aia_2012_pass(self):
         self.assertEqual(res_year, ref_year)
         self.assertEqual(res_val, ref_val)
 
-    def test_gdp_sxm_2012_pass(self):
+    def test_gdp_sxm_2010_pass(self):
         """Test gdp function Sint Maarten."""
-        ref_year = 2012
+        ref_year = 2010
```

Comment:
```suggestion
        # If World Bank input data changes, make sure to set ref_year to a year where
        # no data is available so that the next available data point has to be selected.
        ref_year = 2010
```

Reply:
From my understanding, the ref_year is chosen on purpose in such a way that it's a year where World Bank data is missing in order to test the functionality of selecting the next available data point.

Reply:
yes, that's correct. the additional commit you suggest points that out and is really helpful, thanks